src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -19,6 +19,9 @@
   </PropertyGroup>
   <ItemGroup Label="Build Items">
     <Compile Include="..\..\..\Compilers\Core\Desktop\IVsSQM.cs" />
+    <Compile Include="..\..\..\Compilers\Helpers\ShadowCopyAnalyzerAssemblyLoader.cs">
+      <Link>InternalUtilities\ShadowCopyAnalyzerAssemblyLoader.cs</Link>
```

Comment:
In order to simplify porting csc/vbc/vbcscompiler to CoreCLR the assembly loaders need to live in the host, not in a shared compiler component. Most of these links will go away and be replaced with a CoreCLR-specific loader.